apps/explorer/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -111,7 +111,11 @@ function formatByTransactionKind(
                     category: 'address',
                 },
                 package: {
-                    value: getObjectId(moveCall.package),
+                    // TODO: Simplify after v0.24.0 launched everywhere, when
+                    // moveCall.package is always an ObjectID (a string)
+                    value: typeof moveCall.package === 'string'
```

Comment:
Fixing up Explorer's use of the changing API, cc @pchrysochoidis, @Jordan-Mysten